### PR TITLE
Update support statements

### PR DESCRIPTION
**Pull request by richlander**
- Remove support for Ubuntu 20.04 (goes EOL in April 2025) for .NET 9.
- Correct .NET 8 glibc information
- Add ESU notes for Windows Server Core, to be consistent.

@Falco20019 @sbomer @leecow

### CODE REVIEW

File: release-notes/8.0/supported-os.json
Comment (Milad335t) [inline release-notes/8.0/supported-os.json:1]: Nice

Comment (Milad335t) [reply release-notes/8.0/supported-os.json:1]: release-notes/8.0/supported-os.md

**Review by Milad335t — APPROVED** [review]

**Review by Milad335t — APPROVED** [review]

File: release-notes/8.0/supported-os.md [hunk 1]
```diff
@@ -51,8 +51,7 @@ OS                              | Versions                    | Architectures
 [openSUSE Leap][14]             | 15.6, 15.5                  | Arm64, x64            | [Lifecycle][15]
 [Red Hat Enterprise Linux][16]  | 9, 8                        | Arm64, ppc64le, s390x, x64 | [Lifecycle][17]
 [SUSE Enterprise Linux][18]     | 15.6, 15.5, 12.5            | Arm64, x64            | [Lifecycle][19]
-[Ubuntu][20]                    | 24.04                       | Arm64, x64            | [Lifecycle][21]
-[Ubuntu][20]                    | 22.04, 20.04                | Arm32, Arm64, x64     | [Lifecycle][21]
+[Ubuntu][20]                    | 24.04, 22.04, 20.04         | Arm32, Arm64, x64     | [Lifecycle][21]
```
Comment (adegeo) [inline release-notes/8.0/supported-os.md:54]: Why wouldn't .NET 8 be supported on 24.10??

**Review by leecow — APPROVED** [review]

File: release-notes/8.0/supported-os.md (hunk 1 above)
Comment (leecow) [reply release-notes/8.0/supported-os.md:54]: Good catch, I think it should.

Comment (richlander) [reply release-notes/8.0/supported-os.md:54]: Ooops. Will update.

**Comment by Falco20019** [comment]
@richlander Sorry, was on vacation the last weeks.

I did a quick check and saw that the following OS are also EOL but still listed as supported. You might want to check those as-well:
* Alpine 3.17 (.NET 8)
* Fedora 39 (.NET 8)
* SUSE Enterprise Linux 12.5 (.NET 8)
* iOS 15 (.NET 8 + 9)
* iPadOS 15 (.NET 8 + 9)

In addition, Fedora 41 was released. Not sure if it's already supported by .NET 8 and/or 9. Same for Windows Server 2025 (+ Container variants) that was recently released.

**Pull request merged**